src/main/java/kenigsberg/wordledictionary/WordleGame.java

```diff
@@ -0,0 +1,74 @@
+package kenigsberg.wordledictionary;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Random;
+
+
+public class WordleGame
+{
+    private final String correctWord;
+    private final Random random = new Random();
+
+    public WordleGame() throws IOException
+    {
+        WordleDictionary wordleDictionary = new WordleDictionary();
+        ArrayList<String> fiveLetterWords = new ArrayList<>();
+
+        ArrayList<String> allWords = wordleDictionary.getList();
+
+
+        for (int i = 0; i < allWords.size(); i++)
+        {
+            if (allWords.get(i).length() == 5)
+            {
+                fiveLetterWords.add(allWords.get(i));
+            }
+        }
+
+        int indexOfCorrectWord = random.nextInt(fiveLetterWords.size());
+        this.correctWord = fiveLetterWords.get(indexOfCorrectWord);
+    }
+
+    public String getCorrectWord()
+    {
+        return correctWord;
+    }
+
+
+    public CharResult[] guess(String guessString)
+    {
+        if (guessString.length() != 5)
+        {
+            return null;
+        }
```

Comment:
Returning null here probably isn't a good idea because it could lead to NullPointerExceptions down the road.
Also I could decide next week that I want you doing Wordle with 6 letters not 5.